src/components/v2/chartDeploymentHistory/chartDeploymentHistory.service.ts

```diff
@@ -0,0 +1,66 @@
+import { Routes } from '../../../config'
+import { get, put } from '../../../services/api'
+import { ResponseType } from '../../../services/service.types'
+import { ActionResponse, ChartMetadata, DeployedAt, InstalledAppInfo } from '../../external-apps/ExternalAppService'
+
+export interface ChartDeploymentManifestDetail {
```

Comment:
Why are we specifying the type in service component instead of separate file

Reply:
As discussed, there's no specific reason. It's just that these types will be used by service for request/response and component (tsx) file inside the same folder and also there're only a couple of types so creating a new *.type.ts file wouldn't be that much of use.